### ISSUE TITLE
[Solved] Official Repo

### ISSUE THREAD

**Issue by coolaj86** [Daplie/node-greenlock#75]
The actual home of Greenlock is https://git.coolaj86.com/coolaj86/greenlock-express.js

This fork is not maintained.

To avoid confusion I have asked the owners of this fork to transfer the repository to me. Others have also asked for this fork to be archived and to have the readme updated.

**Comment by styfle** [comment]
That page doesn't load for me but I found this: https://github.com/coolaj86/greenlock

**Comment by coolaj86** [comment]
@styfle Can you check and see if it is loading for you now?

I have an uptime monitor on it and got a notification last night of a 1 minute outage (probably Digital Ocean rebooting for the L1TF patch), but haven't seen any problems other than that (and someone even opened a new issue).

**Comment by styfle** [comment]
> Are you behind any sort of network system that might be blocking sites

Yes my workplace blocks some websites. Usually it's when the IP is outside of North America, but yours seems to be in San Francisco so I'm not sure why it's blocked.

**Comment by coolaj86** [comment]
IPs are bought and sold all of the time. I switched to a new server last month. This month it belongs to me, last month it could have been a blacklisted Russian mail server.
